jte-maven-plugin/src/main/java/gg/jte/maven/GeneratorMojo.java

```diff
@@ -77,7 +83,13 @@ public void execute() {
             templateEngine.setGenerateNativeImageResources(generateNativeImageResources);
         }
         templateEngine.setProjectNamespace(project.getGroupId() + "/" + project.getArtifactId());
-
+        getLog().info("extensions=" + extensions);
```

Comment:
Should we move this into the if, so that it only logs if there are extensions configured?

Reply:
Yes, will do.